### ISSUE TITLE
abnormal circulating anti-Mullerian hormone level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#4014]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  abnormal circulating anti-Mullerian hormone level
2.  decreased circulating anti-Mullerian hormone level
3.  increased circulating anti-Mullerian hormone level 

>2. Synonyms (e.g., absent spleen)

1.  aberrant circulating anti-Mullerian hormone level
2.  reduced circulating anti-Mullerian hormone level
3.  elevated circulating anti-Mullerian hormone level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. This is an aberrant concentration of anti-Mullerian hormone in the blood.  AMH is a hormone that helps form reproductive organs in males and females.
2.  This is an abnormally low concentration of anti-Mullerian hormone in the blood.  AMH is a hormone that helps form reproductive organs in males and females.
3.  This is an abnormally high concentration of anti-Mullerian hormone in the blood.  AMH is a hormone that helps form reproductive organs in males and females.

PMID:34256052 

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal circulating hormone level (MP:0005418)
2. abnormal circulating anti-Mullerian hormone level (new term)
3. abnormal circulating anti-Mullerian hormone level (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by sbello** [comment]
Added
Abnormal (Class(<http://purl.obolibrary.org/obo/MP_0014513>))
Increased (Class(<http://purl.obolibrary.org/obo/MP_0014514>))
Decreased (Class(<http://purl.obolibrary.org/obo/MP_0014515>))

no eq, can't find an entry for this in PRO or CHEBI
